runtime/doc/options.txt

```diff
@@ -5427,6 +5427,13 @@ A jump table for the options with a short description can be found at |Q_op|.
 	bin	If included, numbers starting with "0b" or "0B" will be
 		considered to be binary.  Example: Using CTRL-X on
 		"0b1000" subtracts one, resulting in "0b0111".
+	unsigned    If included, numbers is recognized to be unsigned number.
+		Thus the numbers does not contained negative sign('-').
```

Comment:
```suggestion
		Thus the numbers does not contained negative sign ('-').
```

Reply:
I think better would be something like:

`If included, numbers are recognized as unsigned. Thus a leading negative sign won't be considered as part of the number.`